mantidproject/mantid
# Fix dead links in documentation

This is for trac ticket [#11680](http://trac.mantidproject.org/mantid/ticket/11680).

This pull request either fixes or removes most of the broken links in the documentation.

**Testing**:
- Build the documentation
- Go to the pages affected
- Check the links work
